app/authentication/auth.py

```diff
@@ -36,12 +44,15 @@ def get_auth_token(req):
     if not auth_header:
         raise AuthError('Unauthorized, authentication token must be provided', 401)
 
-    auth_scheme = auth_header[:7].title()
-
-    if auth_scheme != 'Bearer ':
-        raise AuthError('Unauthorized, authentication bearer scheme must be used', 401)
+    for el in AUTH_TYPES:
+        scheme, auth_type = el
+        if auth_header.lower().startswith(scheme.lower()):
+            token = auth_header[len(scheme) + 1:]
+            return auth_type, token
 
-    return auth_header[7:]
+    # TODO: decide if error message stays the same.
+    # In fact we support another auth method than Bearer
+    raise AuthError('Unauthorized, authentication bearer scheme must be used', 401)
```

Comment:
Not sure if we want to keep this exact message or announce that we support other auth types here

Reply:
This would be a good @willeybryan question

Reply:
When would this message trigger if we remove the requirement to use it?

Reply:
This message is displayed when:
- the HTTP header `Authorization` is not set
- you put the HTTP header `Authorization` but not a supported auth type like `Bearer`  and `ApiKey-v1`. These schemes are not case-sensitive

Reply:
Ah so,  AuthError, could respond with ('Unauthorized, authentication bearer scheme or apikey-v1 must be used', 401) and could still be accurate. 

Lets not touch that yet, I am not prepared to answer questions about it yet. Make sure we circle back when we launch it with content.

Reply:
Yes, that would be the correct error message in the future. But I thought it was good to hold from updating it for now and we want to silently launch it yet.